diff --git a/ui/build/paths/config.go b/ui/build/paths/config.go
index b3092ea..7e6d155 100644
--- a/ui/build/paths/config.go
+++ b/ui/build/paths/config.go
@@ -107,6 +107,8 @@ var Configuration = map[string]PathConfig{
 	"tr":      Allowed,
 	"unzip":   Allowed,
 	"zip":     Allowed,
+	"make":    Allowed,
+	"nproc":   Allowed,
 
 	// Host toolchain is removed. In-tree toolchain should be used instead.
 	// GCC also can't find cc1 with this implementation.
